ckanext/dbquery/tests/test_blueprints.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+import ckan.tests.helpers as helpers
+from bs4 import BeautifulSoup
+
+
+@pytest.mark.usefixtures("clean_db")
+@pytest.mark.ckan_config("ckan.plugins", "dbquery")
+@pytest.mark.usefixtures("with_plugins")
+class TestDBQueryBlueprints:
+
+    def test_index_not_authorized(self, app, normal_user):
+        """Test that non-sysadmin users can't access the index page."""
+        helpers.call_action('user_create', name=normal_user['name'],
+                            email=normal_user['email'], password='password')
+
+        with app.flask_app.test_client() as client:
+            # Login as normal user
+            client.post('/user/login', data={'login': normal_user['name'], 'password': 'password'})
+
+            # Try to access the DB query page
+            response = client.get('/ckan-admin/db-query/')
+            assert response.status_code == 403
```

Comment:
```suggestion
        response = client.get('/ckan-admin/db-query/', headers=...
        assert response.status_code == 403
```

No entiendo porque todo esto, en otros tests lo resolvemosdirectamente con el header en el request, ver tests de  otras extensiones como `announcements` o `api-tracking`